### PR TITLE
navbar default property

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**

### CODE REVIEW

File: src/taipy/gui/viselements.json [hunk 1]
```diff
@@ -758,8 +758,9 @@
           "properties": [
             {
                 "name": "lov",
+                "default_property": true,
                 "type": "dict[str, any]",
```
Comment (FabienLelaquais) [inline src/taipy/gui/viselements.json:762]: Why isn't this `dict[str, str]`, reading the doc?

**Review by FabienLelaquais — APPROVED** [review]
👌

File: src/taipy/gui/viselements.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply src/taipy/gui/viselements.json:762]: its str or icon + str ...

Comment (FabienLelaquais) [reply src/taipy/gui/viselements.json:762]: Ok so the doc is, say incomplete...

Comment (FredLL-Avaiga) [reply src/taipy/gui/viselements.json:762]: same as lovComp.lov

**Pull request merged**